both/collections/round.js

```diff
@@ -30,6 +30,13 @@ _.extend(Round.prototype, {
     assert(this.eventCode);
     return wca.eventByCode[this.eventCode].solveTimeFields;
   },
+  roundCode: function() {
+    if(this.isLast()) {
+      return this.softCutoff ? 'c' : 'f';
+    } else {
+      return (this.softCutoff ? 'degc' : '123f')[this.nthRound-1];
```

Comment:
OK. That makes a lot of sense. And maybe the WCA regulations part and the import/export should be separate things.